win32/CMakeLists.txt

```diff
@@ -59,9 +59,9 @@ if (WIN32)
       DESTINATION ${CMAKE_INSTALL_BINDIR}
       COMPONENT binaries
       )
-  elseif (EXISTS ${CMAKE_FIND_ROOT_PATH}/lib/libbiblesync.dll)
+  elseif (EXISTS ${CMAKE_FIND_ROOT_PATH}/bin/libbiblesync.dll)
```

Comment:
This doesn't smell right. A dll in an a bin directory instead of a lib directory?

Then again Windows is a bad smell, so if I'm just out of touch ... carry on.

Reply:
All .dll files live in the bin directory for Windows stuff. Windows expects to find the dll files either alongside the file that's importing them or in C:\Windows\system32. There aren't really any other path options. So the mingw framework on Linux tends to do the same. DLL files are placed in ${sys_root}/bin and the .dll.a files (the stubs needed by gcc to link against the DLL) are placed in ${sys_root}/lib instead (there's no way to just point at a DLL and link against it, directly, the way there is for a .so file in Linux). When you distribute to the end user, you package up the stuff in /bin/ and leave the stuff in /lib/ for development installs.